react/src/components/atoms/forms/InputCheckBox/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Input from '../Input';
+import Error from '../Input/error';
+import Icon from '../../icons/Icon';
+import { InputContext } from '../Input/context';
+import './style.css';
+
+const CheckBox = (props) => (
+  <React.Fragment>
+    <InputContext.Consumer>
+      {
+        (context) => {
+          const { id, value } = context;
+          const {
+            icon, label, disabled, required
+          } = props;
+          const handleClick = () => {
+            context.updateState({ value: !value }, () => {
+              if (typeof props.onChange === 'function') {
+                props.onChange(context.getValue(), id);
+              }
+            });
+            if (!!value && required) {
+              context.updateState({ showError: true });
+            } else {
+              context.updateState({ showError: false });
+            }
+          };
+          return(
+            <button
+              className="ma__input-checkbox"
+              onClick={handleClick}
+              disabled={disabled}
+            >
+              <input type="checkbox" id={id} checked={value} readOnly />
```

Comment:
shouldnt we add required here?
```suggestion
              <input type="checkbox" id={id} checked={value} readOnly required={this.props.required} />
```